Dockerfile

```diff
@@ -12,23 +12,28 @@ LABEL io.k8s.description="Archival code service" \
       io.openshift.min-memory="64Mi"
 
 WORKDIR /home/repour
-ENTRYPOINT ["./setup.sh", "-", "./pid1.py", "./au.py", "python3", "-m", "repour"]
+ENTRYPOINT ["sudo", "-E", "-u", "repour", "./pid1.py", "./au.py", "python3", "-m", "repour"]
```

Comment:
This isn't right for a few reasons (`./setup.sh` had these same problems).

First, `sudo`/`su` doesn't work in OSE, as it forces an arbitrary initial UID (you're never root). If I run the [integration tests](https://github.com/project-ncl/repour#integration-tests) I get:

    sudo: unknown uid 1000: who are you?

at the end of the Container Startup Logs. `1000` being my UID, which the integration tests use when simulating OSE UID-forcing.

Second, `pid1.py` must be the first thing executed, as its name indicates. If it's not the first process in the container, it can't reap the zombie processes `ssh` creates, and eventually the container will stop functioning, due to no available PIDs. Anything not specifically written to reap children (usually only systemd or other init systems) won't do it, so `sudo`, `bash`, etc. aren't suitable.

Third, having any process in the container but repour itself is problematic from a Docker perspective (Normally I wouldn't have any, but OSE forced me to, as I [explain in the readme](https://github.com/project-ncl/repour#docker-images-and-open-shift)). One of these problems is making sure signals are forwarded through the wrapper programs, otherwise the container can't be terminated cleanly. This is very hard to do properly in bash, which is why both `pid1.py` and `au.py` are carefully written Python programs. Maybe `sudo` will do this, but I haven't checked.